src/gui/accountsettings.cpp

```diff
@@ -571,7 +571,7 @@ void AccountSettings::slotAccountStateChanged(int state)
         } else if (state == AccountState::SignedOut) {
             showConnectionLabel( tr("Signed out from %1.").arg(serverWithUser) );
         } else {
-            showConnectionLabel(tr("No connection to %1 at %2.")
```

Comment:
Hi,
I removed the dot because it is re-added by the other modification I made (the join adds the dot).
However based on phil-davis comment above and the discussions we are having on the French translation team, I understand this is not a satisfying PR, so I think I should update it (more below in the discussion thread)